services/thermal_mgr/thermal_mgr.c

```diff
@@ -44,17 +45,56 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
   /* Send an event to the thermal manager queue */
 
+  if (thermalMgrQueueHandle == NULL || event == NULL) {
+    return ERR_CODE_INVALID_ARG;
+  }
+
+  xQueueSend(thermalMgrQueueHandle, event, (TickType_t) 0);
+
   return ERR_CODE_SUCCESS;
 }
 
 void osHandlerLM75BD(void) {
   /* Implement this function */
+  thermal_mgr_event_t osEvent = {0};
+  osEvent.type = THERMAL_MGR_EVENT_OS_CMD;
+
+  thermalMgrSendEvent(&osEvent);
 }
 
-static void thermalMgr(void *pvParameters) {
+static void thermalMgr(void *pvParameters) { // what is pvParameters used for?
```

Comment:
This is just a comment, you can resolve this once you read it. `pvParameters` isn't used but i can be used to send values to the task on start at runtime that can't be done via macros at compile time